### ISSUE TITLE
UWD doesn't work at all on latest insider canary versions

### ISSUE THREAD

**Issue by machineonamission** [pr701/universal-watermark-disabler#2]
Currently on
```
Edition	Windows 11 Home Insider Preview
Version	23H2
Installed on	‎2024-‎01-‎04
OS build	26020.1000
Experience	Windows Feature Experience Pack 1000.26020.1000.0
```
but it's been not working for a while, i only just found this repo to report the issue.

**Comment by pockyneko** [comment]
same problem,can dear author fix it

**Comment by machineonamission** [comment]
I’m working on a replacement for uwd called uwd2. It’s not officially released yet but it works if you build it yourself. It should be just as capable, but more general, less prone to breaking, and no system files modified. It doesn’t even require admin perms!

https://github.com/reticivis-net/uwd2

**Comment by pockyneko** [comment]
> 我正在开发一个名为 uwd2 的 uwd 替代品。它尚未正式发布，但如果您自己构建它就可以使用。它应该同样有能力，但更通用，不易损坏，并且不修改系统文件。它甚至不需要管理员权限！
> 
> https://github.com/reticivis-net/uwd2

thanks,love u ,dude

**Comment by machineonamission** [comment]
UWD2 is officially complete and released!

**Comment by cookie9822** [comment]
hey doesn't work i have always the latest windows 11 Preview canary builds any links for updated version ?

**Comment by machineonamission** [comment]
> hey doesn't work i have always the latest windows 11 Preview canary builds any links for updated version ?

My new program UWD2 works! https://github.com/machineonamission/uwd2

**Comment by cookie9822** [comment]
thank you this works but after every restart of windows the watermark is back do i have to run uwd 2 exe every time after restart?

**Comment by cookie9822** [comment]
after every restart the watermark appears again is that  normal is running the uwd2 exe necessary to run every restart ?

**Comment by machineonamission** [comment]
>  after every restart the watermark appears again is that  normal is running the uwd2 exe necessary to run every restart ?
> 
> 

Yes, that is normal, uwd2 modifies ram not disk so you need to set it as a startup program

**Comment by machineonamission** [comment]
>  after every restart the watermark appears again is that  normal is running the uwd2 exe necessary to run every restart ?
> 
> 

Yes, that is normal, uwd2 modifies ram not disk so you need to set it as a startup program

**Comment by cookie9822** [comment]
ok thanks bro for your time and all effort much appreciate it.
